internal/services/formatters/generic/trivy/formatter_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trivy
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	entitiesAnalysis "github.com/ZupIT/horusec-devkit/pkg/entities/analysis"
+	cliConfig "github.com/ZupIT/horusec/config"
```

Comment:
Can we remove this alias and use just `config.Config`? I think that is better we use a better var name to avoid shadowing instead alias an import. [And imports should be lower case](https://blog.golang.org/package-names#TOC_2.)